engine/src/silo/whitelist.rs

```diff
@@ -0,0 +1,156 @@
+use aurora_engine_sdk::io::{StorageIntermediate, IO};
+use aurora_engine_types::storage::{bytes_to_key, KeyPrefix};
+use aurora_engine_types::AsBytes;
+use borsh::{BorshDeserialize, BorshSerialize};
+
+use crate::prelude::Vec;
+use crate::silo::parameters::{WhitelistKindArgs, WhitelistStatusArgs};
+
+const STATUS: &[u8] = b"LIST_STATUS";
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq, BorshSerialize, BorshDeserialize)]
+pub enum WhitelistKind {
+    Admin = 0x0,
+    EvmAdmin = 0x1,
```

Comment:
Can you document what the difference is between admin and EVM admin? In fact, documenting all of them would be nice.